packages/fast-components-styles-msft/src/radio/index.ts

```diff
@@ -74,7 +74,13 @@ const styles: ComponentStyles<RadioClassNameContract, DesignSystem> = {
         ...applyFocusVisible({
             boxShadow: format<DesignSystem>("0 0 0 1px {0} inset", neutralFocus),
             borderColor: neutralFocus,
+            "@media (-ms-high-contrast:active)": {
+                boxShadow: format<DesignSystem>("0 0 0 2px ButtonText"),
```

Comment:
Any way to keep this inside like non-HC?

Reply:
moved focused inside